cmd/influx/main.go

```diff
@@ -249,6 +258,22 @@ func (c *CommandLine) SetFormat(cmd string) {
 	}
 }
 
+func (c *CommandLine) dump() {
```

Comment:
None of the flag names indicate a type, although the help mesg does it somewhat implicitly.

I wish I had never agreed to creating a special flag for this at all.

I'll use '--dump-the-database'